custom/staking/src/lib.rs

```diff
@@ -1314,6 +1346,62 @@ decl_module! {
             Self::deposit_event(RawEvent::InvalidatedNominators(caller, expired_nominators));
         }
 
+        /// Enables individual commisions. This can be set only once. Once individual commission
+        /// rates are enabled, there's no going back.  Only Governance committee is allowed to
+        /// change this value.
+        #[weight = SimpleDispatchInfo::FixedOperational(100_000)]
+        fn enable_individual_commissions(origin) {
+            T::RequiredCommissionOrigin::try_origin(origin)
+                .map(|_| ())
+                .or_else(ensure_root)
+                .map_err(|_| Error::<T>::NotAuthorised)?;
+
+            // Ensure individual commissions are not already enabled
+            if let Commission::Global(_) = <ValidatorCommission>::get() {
+                <ValidatorCommission>::put(Commission::Individual);
+                Self::deposit_event(RawEvent::IndividualCommissionInEffect);
+            } else {
+                Err(Error::<T>::IndividualCommissionsEnabled)?
```

Comment:
I think Error should be `AlreadyIndividualComissionsEnabled`

Reply:
Do you mean the error should be renamed?

Reply:
Yes